delivery_price_method/tests/test_delivery_price_method.py

```diff
@@ -58,34 +58,35 @@ def _add_delivery(self):
 
     def test_delivery_price_fixed(self):
         sale = self.sale
+        self.pricelist.item_ids[0].write({"fixed_price": 99.99})
```

Comment:
I still don't understand why is this needed, being the rule to use public price. Shouldn't you put instead this price in the product or see if the pricelist item structure is different?